src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, net_params, batch_size, num_classes):
         for key in net_params.keys():
             net_params[key] = _utils.convert_shared_float_array_to_numpy(net_params[key])
 
+        if 'random_seed' in config:
+            seed = int(_utils.convert_shared_float_array_to_numpy(config['random_seed']))
+            _tf.set_random_seed(seed)
```

Comment:
What is the random seed used for in the DC backend?

Reply:
Didn't need this. Removed this and config everywhere.